numalogic/connectors/utils/aws/sts_client_manager.py

```diff
@@ -0,0 +1,97 @@
+import logging
+from datetime import datetime, timezone
+import boto3
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class STSClientManager:
```

Comment:
Can we reuse or merge from registries/dynamodb connector? Don't have to do it now.